changelog/v1.19.0-beta1/cluster-stats-names.yaml

```diff
@@ -0,0 +1,10 @@
+changelog:
+  - type: BREAKING_CHANGE
```

Comment:
i'm not sure if this should be considered breaking or not, given that it only affects stats names

Reply:
Well, dashboards and alerts can break, so I'd say it is a breaking change.